geomstats/geometry/invariant_metric.py

```diff
@@ -55,9 +123,11 @@ def __init__(self, space, metric_mat_at_identity=None, left=True):
         self._instantiate_solvers()
 
     def _instantiate_solvers(self):
```

Comment:
Add docstring, even for a private function, to help readability (we are having problems using the solvers in the lab, because we are having problems understanding the code). In particular, explain the choice of the hardcoded parameters. n_nodes, use_jac, etc.